client/ios/Podfile.lock

```diff
@@ -333,6 +333,6 @@ SPEC CHECKSUMS:
   ReactCommon: 198c7c8d3591f975e5431bec1b0b3b581aa1c5dd
   Yoga: f2a7cd4280bfe2cca5a7aed98ba0eb3d1310f18b
 
-PODFILE CHECKSUM: f96bb4d63a7dac3a82c4aa25eec2f44afc9e0222
+PODFILE CHECKSUM: 26c1dc71a74a5035f07d8cae1aea550eb7ff2701
 
-COCOAPODS: 1.7.2
+COCOAPODS: 1.8.4
```

Comment:
use the same cocoa pods as everyone else in the group otherwise, it will throw errors.

Reply:
I'll make sure everyone to have same the cocoapod version 😚